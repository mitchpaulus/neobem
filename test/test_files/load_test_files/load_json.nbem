zone_data = load({ 'path': 'data.json', 'type': 'JSON' })

zone_template = Î» zone {
Zone,
  < zone.'name' >,
  0,
  < zone.'x origin' >,
  0,
  0,
  1,
  1,
  autocalculate,
  autocalculate,
  autocalculate,
  ,
  ,
  Yes;
}

print map(zone_template, zone_data.'zones')
