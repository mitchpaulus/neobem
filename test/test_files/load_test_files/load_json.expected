Zone,
  Zone 1,
  0,
  10,
  0,
  0,
  1,
  1,
  autocalculate,
  autocalculate,
  autocalculate,
  ,
  ,
  Yes;

Zone,
  Zone 2,
  0,
  5,
  0,
  0,
  1,
  1,
  autocalculate,
  autocalculate,
  autocalculate,
  ,
  ,
  Yes;


